runtime/server/generate_metrics_view.go

```diff
@@ -362,23 +358,19 @@ func generateMetricsViewYAMLSimpleDimensions(schema *runtimev1.StructType) []*me
 func generateMetricsViewYAMLSimpleMeasures(schema *runtimev1.StructType) []*metricsViewMeasureYAML {
 	var measures []*metricsViewMeasureYAML
 	measures = append(measures, &metricsViewMeasureYAML{
-		Name:                "total_records",
-		Label:               "Total records",
-		Expression:          "COUNT(*)",
-		Description:         "",
-		FormatPreset:        "humanize",
-		ValidPercentOfTotal: true,
+		Label:       "Total records",
+		Expression:  "COUNT(*)",
+		Name:        "total_records",
+		Description: "",
 	})
 	for _, f := range schema.Fields {
 		switch f.Type.Code {
 		case runtimev1.Type_CODE_FLOAT32, runtimev1.Type_CODE_FLOAT64:
 			measures = append(measures, &metricsViewMeasureYAML{
-				Name:                f.Name,
-				Label:               fmt.Sprintf("Sum of %s", identifierToTitle(f.Name)),
-				Expression:          fmt.Sprintf("SUM(%s)", safeSQLName(f.Name)),
-				Description:         "",
-				FormatPreset:        "humanize",
-				ValidPercentOfTotal: true,
```

Comment:
I think it makes sense not to include `valid_percent_of_total` by default, but wondering if we should make it evaluate to `true` by default? Since it's true for most cases, and only slightly confusing for the cases where it's not.

If yes, we can make it such that it becomes `false` only if explicitly set to false in the YAML (and `true` if not present in the YAML at all)

Reply:
Its hard to assume its true for all metrics by default as for avg it would give wrong information on the dashboards. We can apply a regex for count and sum on simple metrics to set it to true otherwise treat it as false. Lets do it on the platform side in a separate PR, where we also have the capability to parse the expression and determine true/false.